x-pack/plugins/security_solution/server/lib/detection_engine/reference_rules/query.ts

```diff
@@ -73,7 +73,6 @@ export const createQueryAlertType = (ruleDataClient: RuleDataClient, logger: Log
         };
 
         const alerts = await findAlerts(query);
-        // console.log('alerts', alerts);
```

Comment:
can probably remove this at some point

Reply:
Yea, deleted this in my PR.

Reply:
oh wow this is at a minimum the second time I commented on removing something that was marked as red. I don't know my colors very well lol.